### DISCUSSION TITLE
Pin out locations & LEDs Greaseweazle

### DISCUSSION THREAD

**Discussion by MD-Orion** [keirf/greaseweazle#176] [General]
GREASEWEAZLE

Purchased off of Ebay !
 
My board has no pins at:   Act+ * 5v *   J9 

What is the pad/pin to the left of ACT+ ?
Is it Ground ?

What is the J9 pad, is it ground ?

Is the 5v pad/pin 5v + ?

If attaching LEDs are resistors in line needed ?

How are the LEDs  ?
(1). Two leds to ground ?
(2). One lead to each positive ?

****I have found a _wiring diagram_ listed as Schematic.
I haven't found a schematic to determine the above !****

Can any one help me ?

**Comment by wohali** [comment]
That sounds like the GW F7 Plus?

https://raw.githubusercontent.com/wiki/keirf/greaseweazle/assets/gallery/f7_plus.jpg

3rd party board, see https://github.com/aerobaticant/greaseweazle-F7-Plus and https://github.com/aerobaticant/Greaseweazle-F7-Plus/wiki

**Reply by MD-Orion** [reply]
Thanks for the information !

**Comment by keirf** [comment]
I think there's an eBay seller flogging these without support.

**Reply by MD-Orion** [reply]
Yes, they only supply man link for this information with no information to reference or lead to answers .
However these pages and the help I have received makes a big difference !

**Comment by keirf** [comment]
The wiki link provided above should give all the info you need.

**Reply by MD-Orion** [reply]
Thanks !
